README.md

```diff
@@ -8,7 +8,7 @@ Imports your current Cloud infrastructure to an Infrastructure As Code [Terrafor
 
 At [Cycloid](https://www.cycloid.io/), Infrastructure As Code is in the company DNA since the beginning. To help our new customers adopting this best practice, we decided to build Terracognita to convert an existing infrastructure on Cloud Infrastructure into Terraform code in an automated way, relying on Terraform providers built by the community. We focused on AWS, GCP and Azure but Alibaba, Vmware and Openstack will be the next to be integrated.
 
-We decided to Open Source this tool as we believe that it will help people to adopt IaC in an easy way. Cycloid provides this tool to let people import their infrastructure into [Cycloid's pipelines](https://www.cycloid.io/devops-framework), allow them to generate infrastructure diagram and manage all infra/application life cycle from a single interface.
+We decided to Open Source this tool as we believe that it will help people to adopt IaC in an easy way. Cycloid provides this tool to let people import their infrastructure into [Cycloid's pipelines](https://www.cycloid.io/devops-framework), allow them to generate infrastructure diagram and manage all infra/application life cycle from a single interface. If you are interested in contributing to Terracognita or simply curious about what's next: you can have a look to the public [roadmap](https://github.com/cycloidio/terracognita/issues/118).
```

Comment:
I would potentially give it more emphasis, at least a `\n`.

Reply:
done, I double checked with @NiamhLynch for the location :)